FAQ.md

```diff
@@ -77,6 +77,29 @@ ssh-add
 and if using the `-b` option, ensure that your user has access to `sudo`
 without a password to the `root` account (often done by adding it to the `wheel` group)
 
+## What about the builds that use the `dockerBuild` tag?
+
+In addition to the static build machines which we have, ther are also
+Dockerfiles that are used to build the base images that our build farm uses
+for running docker based builds on some of our platforms - this is what we
+have at the moment:
+
+| Dockerfile | Image | Platforms  | Where is this built? | In use?
+|---|---|---|---|
+| [Centos7](./ansible/Dockerfile.CentOS7) | [`adoptopenjdk/centos7_build_image`](https://hub.docker.com/r/adoptopenjdk/centos7_build_image) | linux/amd64, linux/arm64 | [Travis](.travis.yml) | Yes
+| [Centos6](./ansible/Dockerfile.CentOS6) | [`adoptopenjdk/centos6_build_image`](https://hub.docker.com/r/adoptopenjdk/centos6_build_image)| linux/amd64 | [GH Actions](.github/workflows/build.yml) | Yes
+| [Alpine3](./ansible/Dockerfile.Alpine3) | [`adoptopenjdk/alpine3_build_image`](https://hub.docker.com/r/adoptopenjdk/alpine3_build_image) | linux/amd64 | [GH Actions](.github/workflows/build.yml) | Yes
+| [Centos7](./ansible/Dockerfile.CentOS7) | [`adoptopenjdk/centos7_build_image`](https://hub.docker.com/r/adoptopenjdk/centos7_build_image) | linux/amd64, linux/arm64 | [Travis](.travis.yml) | No
```

Comment:
You've got Centos7 twice in the table - though one of them is apparently not in use ..?